pcdsdevices/mv_interface.py

```diff
@@ -120,3 +134,352 @@ def umvr(self, delta, timeout=None):
             will be use.
         """
         self.umv(delta + self.wm(), timeout=timeout)
+
+
+def setup_preset_paths(**paths):
+    """
+    Prepare the `Presets` class with the correct paths for saving and loading
+    presets.
+
+    Parameters
+    ----------
+    **paths: ``str`` keyword args
+        A mapping from type of preset to destination path. These will be
+        directories that contain the yaml files that define the preset
+        positions.
+    """
+    Presets._paths = {}
+    for k, v in paths.items():
+        Presets._paths[k] = Path(v)
+
+
+class Presets:
+    """
+    Manager for device preset positions. This provides methods for adding new
+    presets, checking which presets are active, and related utilities.
+
+    It will install the ``mv_presetname`` and ``wm_presetname`` methods onto
+    the associated device.
```

Comment:
I think the format for the class docstrings should be one line summary then longer paragraph explanation.

Reply:
Yeah, you're right, that's the standard. I always mess this up on the first pass, I'll go back and fix it.